org.openmw.OpenMW.json

```diff
@@ -0,0 +1,360 @@
+            "sha256": "5dd4ea0c7e97ad8e3677ff3a254b116df08a5d041c2df8859aad5c4f88d1f774"
+          }
+        ],
+        "cleanup": [
+          "/include",
+          "/lib/*.a",
+          "/lib/*.la",
+          "/lib/pkgconfig"
+        ]
+      },
+      {
+        "name": "boost",
+        "sources": [
+          {
+            "type": "archive",
+            "url": "https://downloads.sourceforge.net/project/boost/boost/1.64.0/boost_1_64_0.tar.bz2",
+            "sha256": "7bcc5caace97baa948931d712ea5f37038dbb1c5d89b43ad4def4ed7cb683332"
+          },
+          {
+            "type": "script",
+            "commands": [
+              "./bootstrap.sh \"$@\"",
+              "cat <<EOF >Makefile",
+              "all:",
+              "\t./b2 -j `nproc` ./b2 stage threading=multi link=shared",
+              "",
+              "install:",
+              "\t./b2 install",
+              "EOF"
+            ],
+            "dest-filename": "configure"
+          }
+        ],
+        "cleanup": [
+          "/include",
+          "/lib/*.a",
+          "/lib/*.la",
+          "/lib/pkgconfig",
+          "libboost_atomic*",
+          "libboost_chrono*",
+          "libboost_container*",
+          "libboost_context*",
+          "libboost_coroutine*",
+          "libboost_date_time*",
+          "libboost_fiber*",
+          "libboost_graph*",
+          "libboost_iostreams*",
+          "libboost_locale*",
+          "libboost_log*",
+          "libboost_math*",
+          "libboost_prg_exec_monitor*",
+          "libboost_python*",
+          "libboost_random*",
+          "libboost_regex*",
+          "libboost_serialization*",
+          "libboost_signals*",
+          "libboost_thread*",
+          "libboost_timer*",
+          "libboost_type_erasure*",
+          "libboost_unit_test_framework*",
+          "libboost_wave*",
+          "libboost_wserialization*"
+        ]
+      },
+      {
+        "name": "openmw",
+        "builddir": true,
+        "buildsystem": "cmake",
+        "config-opts": [
+          "-DBUILD_BSATOOL=no",
+          "-DBUILD_ESMTOOL=no",
+          "-DBUILD_MYGUI_PLUGIN=no"
+        ],
+        "sources" : [
+          {
+            "type": "git",
+            "url": "git://github.com/OpenMW/openmw.git",
+            "branch": "openmw-0.42.0"
+          },
+          {
+            "type": "shell",
+            "commands": [
+              "sed -i s:Icon=openmw:Icon=org.openmw.OpenMW.png: files/openmw.desktop",
+              "sed -i s:Icon=openmw-cs:Icon=org.openmw.OpenMW.OpenCS.png: files/openmw-cs.desktop"
+            ]
+          }
+        ],
+        "post-install": [
+          "mv /app/share/applications/openmw.desktop /app/share/applications/org.openmw.OpenMW.desktop",
+          "mv /app/share/applications/openmw-cs.desktop /app/share/applications/org.openmw.OpenMW.OpenCS.desktop",
+
+          "mkdir -p /app/share/appdata",
```

Comment:
is there a reason why you don't use the builtin 'rename-appdata' 'rename-icon' and 'rename-desktop-file' stuff?